daml/Marketplace/Trading/Order.daml

```diff
@@ -0,0 +1,83 @@
+module Marketplace.Trading.Order where
+
+import DA.Finance.Asset (AssetDeposit)
+import DA.Finance.Types (Id(..), Asset(..))
+
+data Side
+    = Buy
+    | Sell
+  deriving (Eq, Show)
+
+data OrderType
```

Comment:
PriceType

Reply:
I'm not convinced PriceType properly reflects Market / Limit / Stop Limit / etc. I did a quick google search and these details relating to an order seem to commonly be referred to an "Order Type".